sparse/usr/sbin/flash-partition

```diff
@@ -146,6 +146,14 @@ if on_device; then
         # partition flashing as it is not needed.
         if test $1 = boot && grep data_subdir= /proc/cmdline &> /dev/null; then
           echo "Skipping flashing of boot partition as multirom setup detected."
+          echo "Update multirom boot image, ramdisk and kernel"
+          pref_path="/android"
+          [[ -d /android/media/0/multirom ]] ||  mkdir -p $pref_path &&  mount /dev/mmcblk0p28 $pref_path
+          mrom_path=$(cat /proc/cmdline | sed 's/.*data_subdir=\(.*\)\/data/\1/')
```

Comment:
how? mrom_check means nothing in this context... i need to know if /android is mounted I am already in mrom_Check true